apis/identity/v1alpha1/iampolicy_types.go

```diff
@@ -37,6 +37,8 @@ type IAMPolicyParameters struct {
 
 	// The name of the policy.
 	Name string `json:"name"`
+
+	Tags []Tag `json:"tags,omitempty"`
```

Comment:
Most of the time there's a doc string. Not sure how helpful this is in this case, but still:

Maybe something like.
```
       // Tags attached to the policy. For more information about
       // tagging, see Tagging customer managed policies (https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html)
       // in the IAM User Guide.
```

Reply:
Sure thing.

Reply:
Added @chlunde , marking this as resolved.